cmd/encryption-v1.go

```diff
@@ -46,6 +47,7 @@ var (
 	errEncryptedObject      = errors.New("The object was stored using a form of SSE")
 	errInvalidSSEParameters = errors.New("The SSE-C key for key-rotation is not correct") // special access denied
 	errKMSNotConfigured     = errors.New("KMS not configured for a server side encrypted object")
+	errKMSKeyNotFound       = errors.New("Invalid KMS keyId")
```

Comment:
```suggestion
	errKMSKeyNotFound       = errors.New("Invalid KMS key ID")
```
.. unless this is dictated by something else.